sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-storage-resource/README.adoc

```diff
@@ -0,0 +1,113 @@
+= Spring Cloud Azure Storage Starter Sample
```

Comment:
This sample was deleted in the migration to the SDK repository. It does, however, serve to demonstrated storage access with environment variables. 

**Note:** auto-create does not work without drastic and impactful refactoring decoupling resource management from the legacy Azure resource management library and from the spring integrations with which it's tightly entangled (Event Hub, Service Bus, etc).